src/Domain/Amr/Helpers.hpp

```diff
@@ -58,4 +58,11 @@ boost::rational<size_t> fraction_of_block_volume(
 template <size_t VolumeDim>
 bool has_potential_sibling(const ElementId<VolumeDim>& element_id,
                            const Direction<VolumeDim>& direction);
+
+/// \ingroup AmrGroup
+/// \brief The ElementId of the parent of the Element with `element_id` with
+/// refinement `flags`
```

Comment:
Could you make clearer here how the `flags` relate to the `element_id`? First I thought that `element_id` is the result of applying the `flags` to the `id_of_parent`, so it's a sort of inverse operation, but now I think I have figured out that `id_of_parent` is the result of joining elements with the `element_id`.